src/atom.coffee

```diff
@@ -572,6 +581,13 @@ class Atom extends Model
     dimensions = @getWindowDimensions()
     @state.windowDimensions = dimensions if @isValidDimensions(dimensions)
 
+  storeWindowBackground: ->
```

Comment:
Would this also resolve https://github.com/atom/atom/issues/929? (Seems like it would.)